datahub-web-react/src/app/entity/shared/components/styled/Join/CreateJoinModal.tsx

```diff
@@ -0,0 +1,341 @@
+            ),
+            dataIndex: 'field2Name',
+            tableRecord: table2DatasetSchema || {},
+            editable: true,
+        },
+        {
+            title: 'Action',
+            dataIndex: '',
+            editable: false,
+            render: (record) =>
+                tableData.length > 1 ? (
+                    <Button type="link" onClick={() => handleDelete(record)}>
+                        Delete
+                    </Button>
+                ) : null,
+        },
+    ];
+    const handleSave = (row: JoinDataType) => {
+        const newData = [...tableData];
+        const index = newData.findIndex((item) => row.key === item.key);
+        const item = newData[index];
+        newData.splice(index, 1, {
+            ...item,
+            ...row,
+        });
+        setTableData(newData);
+    };
+    const components = {
+        body: {
+            row: EditableRow,
+            cell: EditableCell,
+        },
+    };
+
+    const columns = defaultColumns.map((col) => {
+        if (!col.editable) {
+            return col;
+        }
+        return {
+            ...col,
+            onCell: (record: JoinDataType) => ({
+                record,
+                editable: col.editable,
+                dataIndex: col.dataIndex,
+                tableRecord: col.tableRecord,
+                title: col.title,
+                handleSave,
+            }),
+        };
+    });
+    return (
+        <Modal
+            title={
+                <div className="footer-parent-div">
+                    <p className="join-title">Join parameters</p>
+                    <div>
+                        <Button onClick={onCancelSelect} className="cancel-btn" size="large">
+                            Cancel
+                        </Button>
+                    </div>
+                    <div>
+                        <Button className="submit-btn" size="large" id="continueButton" onClick={onSubmit}>
+                            Submit
+                        </Button>
+                    </div>
+                </div>
+            }
+            visible={visible}
+            closable={false}
+            className="CreateJoinModal"
+            okButtonProps={{ hidden: true }}
+            cancelButtonProps={{ hidden: true }}
+            onCancel={onCancelSelect}
+            destroyOnClose
+        >
+            <div className="inner-div">
+                <p className="all-table-heading">Table 1</p>
+                <p className="all-information">{table1NameBusiness}</p>
+                <div className="techNameDisplay">{table1NameTech !== table1NameBusiness && table1NameTech}</div>
+                <p className="all-content-heading">Table 2</p>
+                <p className="all-information">{table2NameBusiness}</p>
+                <div className="techNameDisplay">{table2NameTech !== table2NameBusiness && table2NameTech}</div>
+                <p className="all-content-heading">Join name</p>
```

Comment:
That was a significant change for us so we skipped time being but please let us know in case you need us to change all places with styled component only or specific areas.

Reply:
Thank you @chriscollins3456 for your time and review it throughly.
Could you please suggest on the above comment?